src/js/merkle-drop/ClaimFlow.js

```diff
@@ -150,6 +147,14 @@ function ClaimFlow() {
     setShowTermsAndConditionsModal(false)
   }
 
+  useEffect(() => {
+    if (!address || !account) {
```

Comment:
This does not work. If account is undefined, this just means that we dont know if there are equal.